.github/workflows/spot-common-workflow.yml

```diff
@@ -0,0 +1,11 @@
+
+name: Common Spot Verification Workflow
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  call-verification-workflow:
+    uses: spotinst/devops-scripts/.github/workflows/common-verification-workflow.yml@master
+    secrets: inherit
```

Comment:
I think we should probably remove this workflow since if people create PRs then they are not going to have access to our jira.